src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/PractitionerMapperTest.java

```diff
@@ -35,7 +34,14 @@
     void testMapMessageToRequestingPractitionerWithNoRequester() {
         when(message.getInvolvedParties()).thenReturn(Collections.emptyList());
 
-        assertThat(mapper.mapToRequestingPractitioner(message)).isEmpty();
+        Practitioner requestingPractitioner = mapper.mapToRequestingPractitioner(message);
+
+        assertAll(
+            () -> assertThat(requestingPractitioner.getName())
+                .hasSize(0),
+            () -> assertThat(requestingPractitioner.getIdentifier())
+                .hasSize(0)
```

Comment:
`isEmpty` might be clearer
```suggestion
            () -> assertThat(requestingPractitioner.getName())
                .isEmpty(),
            () -> assertThat(requestingPractitioner.getIdentifier())
                .isEmpty()
```